<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImagePreferences</class>
 <widget class="QDialog" name="ImagePreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="aspect_ratio_text">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Aspect Ratio:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="aspect_ratio">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Keep Aspect Ratio</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16 : 10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16 : 9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5 : 4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4 : 3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3 : 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1 : 1</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="gray_scale_cb">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::RightToLeft</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {
    width: 20px;
    height: 20px;
	border: 1px solid grey;
	border-radius: 4px;
	color: white;
}

QCheckBox::indicator:checked {
	background: #32a6ff;
}</string>
   </property>
   <property name="text">
    <string>Gray Scale                      </string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="remove_alpha_cb">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {
    width: 20px;
    height: 20px;
	border: 1px solid grey;
	border-radius: 4px;
	color: white;
}

QCheckBox::indicator:checked {
	background: #32a6ff;
}</string>
   </property>
   <property name="text">
    <string>Remove Alpha                </string>
   </property>
  </widget>
  <widget class="QCheckBox" name="quality_cb">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {
    width: 20px;
    height: 20px;
	border: 1px solid grey;
	border-radius: 4px;
	color: white;
}

QCheckBox::indicator:checked {
	background: #32a6ff;
}</string>
   </property>
   <property name="text">
    <string>Quality                           </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="quality_range">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="bit_depth_cb">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {
    width: 20px;
    height: 20px;
	border: 1px solid grey;
	border-radius: 4px;
	color: white;
}

QCheckBox::indicator:checked {
	background: #32a6ff;
}bom</string>
   </property>
   <property name="text">
    <string>Bit Depth                         </string>
   </property>
  </widget>
  <widget class="QComboBox" name="bit_depth">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>...</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>24</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>32</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="save_image_preferences">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Preferences</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_image_preferences">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel Changes</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
